axelor-sale/src/main/java/com/axelor/apps/sale/rest/SaleOrderRestController.java

```diff
@@ -41,6 +43,7 @@
 @Consumes(MediaType.APPLICATION_JSON)
 @Produces(MediaType.APPLICATION_JSON)
 public class SaleOrderRestController {
+
   @Operation(
       summary = "Create a sale oder",
       tags = {"Sale order"})
```

Comment:
We need to change SecurityCheck and add writeAccess for SaleOrder and createAccess for SaleOrderLine